src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-card flat id="upload-memorandum">
+
+    <!-- Summary Section -->
+    <div id="upload-memorandum-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="upload-memorandum-summary-header" >
+        <v-icon color="#38598A">mdi-text-box-multiple</v-icon>
+        <label class="upload-memorandum-title"><strong>Memorandum</strong></label>
+      </div>
+
+      <!-- Summary error message -->
+      <div v-if="!valid" class="upload-memorandum-error-message">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text"> This step is not complete. </span>
+          <router-link :to="{ path: `/${RouteNames.CREATE_MEMORANDUM}`, query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <v-card-title>Memorandum Placeholder component</v-card-title>
+  </v-card>
+</template>
+<script lang="ts">
+// Libraries
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { State } from 'vuex-class'
+
+// Enums
+import { RouteNames } from '@/enums'
+
+@Component({})
+export default class UploadMemorandum extends Vue {
+  // Global state
+  @State(state => state.stateModel.createRulesStep.valid)
```

Comment:
I'd prefer we use getters instead of accessing state.

I searched and found that accessing state is done a lot in this project (gack!).

Reason: we don't want components to assume (or think they know) what the state structure looks like; we do want components to use an "API" (interface/contract) between system modules (like store).

Reply:
For sure, i agree.
Changing everywhere is the right thing to do, but the scope of that change is a little larger and will require regression testing. 

For this PR, i will update these new/edited instances of state accessors to getters and we should make effort to update these as we go.

Reply:
Excellent, thanks. Rome wasn't built in a day.

Reply:
At least 2 days!

Reply:
Updated in next commit 👍